.appveyor.yml

```diff
@@ -1,13 +1,12 @@
 environment:
   matrix:
-  - nodejs_version: "6.9"
+    - nodejs_version: "6"
```

Comment:
We actually really need 6.9 minimum because of protractor. `.nvmrc` should be changed instead to reflect that.